dist/js/fontawesome-iconpicker.js

```diff
@@ -956,24 +981,28 @@ module.exports = allIconNames;
                 return a(false);
             } else {
                 var d = [];
+                var e = [];
+                var f = [];
+                var g = false;
+                try {
+                    g = new RegExp("(?:^|\\W)" + c);
+                } catch (a) {
+                    g = false;
+                }
                 this.iconpicker.find(".iconpicker-item").each(function() {
                     var b = a(this);
-                    var e = b.attr("title").toLowerCase();
-                    var f = b.attr("data-search-terms") ? b.attr("data-search-terms").toLowerCase() : "";
-                    e = e + " " + f;
-                    var g = false;
-                    try {
-                        g = new RegExp("(^|\\W)" + c, "g");
-                    } catch (a) {
-                        g = false;
-                    }
-                    if (g !== false && e.match(g)) {
+                    var c = b.attr("title").toLowerCase();
+                    var h = b.attr("data-search-terms") ? b.attr("data-search-terms").toLowerCase() : "";
+                    c = c + " " + h;
```

Comment:
You can declare c directly here and avoid reassigning the variable

Reply:
That's compiled code :smile:, the distribution is intended to be manually compiled for now and included in the commits.

The corresponding code was not modified - yeah the reassignment is ugly, but was outside the scope of the fix.

I'd like to make as few as possible changes, so we can make an upstream PR.

Reply:
Ah sorry wrong file :P
I thought we were going to keep our fork, but if we plan on switching to using the upstream keep it as it is :)

Reply:
No, actually we *will* keep our fork, it's just that it we're still trying to contribute back to upstream (check the [commits](https://github.com/itsjavi/fontawesome-iconpicker/commits/master)), it will make things easier for us if/when we decide to upgrade.